src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
The class `Figure` is using an abstract class to set behavior by implementing both `AreaCalculator` and `Drawable` interfaces. According to the checklist, abstract classes should not be used to set behavior. Consider using interfaces to define behaviors instead.